docs/user/openstack/vgpu.md

```diff
@@ -0,0 +1,456 @@
+# Installing a cluster on OpenStack with vGPU support
+
+If the underlying OpenStack deployment have proper GPU hardware installed and
+configured there is a way for passing down vGPU to the pods by using
+gpu-operator.
```

Comment:
Looking at docs [1] they name it GPU Operator?

[1] https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/contents.html

Reply:
Yup.